composer/README.md

```diff
@@ -0,0 +1,74 @@
+# Set up composer
+
+Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you.
+You will need composer for any of Symfony based project.
+
+## 1 Install
+
+Installation procedure should be the same for MacOS and Linux. You can find detailed instructions on https://getcomposer.org/download/ and https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos
+
+By default, the above instructions will install the **latest** composer version.
+As composer2 is not (yet) used on most projects, we will first install composer1 and make it available globally:
+```console
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+php -r "if (hash_file('sha384', 'composer-setup.php') === '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+php composer-setup.php --version=1.10.21
+php -r "unlink('composer-setup.php');"
+```
+
+Next step is making sure `composer` command is available globally:
+```console
+mv composer.phar /usr/local/bin/composer
```

Comment:
I would recommend moving the Composer binary to `~/bin/` instead of `/usr/local/bin` as it makes it possible to update Composer binary without `sudo`. `~/bin/` should be in path ofcourse, but I believe, at least on Linux, it already is?

Reply:
It does not seem to be in the path by default?
`/home/iherak/.local/bin` seems to be there though, so maybe we could use that one (same one is suggested on https://getcomposer.org/doc/00-intro.md#globally)

Reply:
Sure :+1: Now that I think of it, maybe I did add ~/bin/ to path myself :)

Reply:
@pspanja is `~/.local/bin` available on MacOS by default?

Reply:
Seems like on MacOS, by default there is `~/bin` :facepalm:

Reply:
It isn't, and neither is `~/bin`, I also had to add it myself.

Reply:
Ok, so keep it as is to make it simpler? I do not think using `sudo` for this should be a problem?

Reply:
It's not a problem, I just wanted to make it easier a bit, but it's obviously a rabbit's hole :D